README.md

```diff
@@ -8,18 +8,13 @@ This is the front-end application for:
 
 * [Trade Tariff Backend](https://github.com/trade-tariff/trade-tariff-backend)
 
-This application requires one or more Trade Tariff Backend APIs to be running and the following env variables set: `API_SERVICE_BACKEND_DEFAULT`, `API_SERVICE_BACKEND_URL_OPTIONS`.
+To run TTF (Trade Tariff Frontend) you need:
 
-For example:
+- to run Trade Tariff Backend APIs
+- to set `API_SERVICE_BACKEND_URL_OPTIONS`
```

Comment:
These are configured by default so we treat this as convention over configuration and leave it undocumented to avoid confusion?

Reply:
Good point, maybe we can just mention it as an important variable, but we does not need to be changed in order to run the FE (and it looks like now)

Reply:
Yeah. This is actually misleading because the reality is by convention we run:

1. The duty calculator on port 3002
2. The frontend on 3001
3. The backend on 3000

So all of these links would be required to point at 3000 anyway.